website/settings/defaults.py

```diff
@@ -158,7 +158,8 @@ def parent_dir(path):
 SHORT_DOMAIN = 'osf.io'
 
 # TODO: Combine Python and JavaScript config
-COMMENT_MAXLENGTH = 500
+# Add 500 to account for variable length due to at mentions
+COMMENT_MAXLENGTH = 500 + 500
```

Comment:
Well the user would be really confused about what is being checked since they don't see the HTML or Markdown version of the comment.